source/isaaclab/config/extension.toml

```
@@ -1,7 +1,7 @@
 [package]
 
 # Note: Semantic Versioning is used: https://semver.org/
-version = "0.36.2"
+version = "0.36.3"
```

We don't need to update the changelog for non-code changes :)

```suggestion
version = "0.36.2"
```

---

got it